base_ubl/models/ubl.py

```diff
@@ -326,8 +326,8 @@ def _ubl_add_line_item(
     @api.model
     def _ubl_add_item(
             self, name, product, parent_node, ns, type='purchase',
-            seller=False, version='2.1'):
-        '''Beware that product may be False (in particular on invoices)'''
+            seller=None, version='2.1'):
```

Comment:
Why change this? This is not a reference variable

Reply:
Well, it should be an empty record of `self.env['product.supplierinfo']` to be precise

Reply:
But then `None` and `False` is the same, and you can't put that reference variable as you can imagine. The point is to not touch here if there's no better option.

Reply:
What problem do you have with `None`? Why should it be `False`? Both functionally are the same. I just preferred `None` because this way in line 340, in `seller.id`, I don't receive stupid PyCharm message of "Unresolved attribute reference 'id' for class 'bool'".

Reply:
I don't have any problem with `None`. The problem is to change things that are not needed. You don't know if dependent modules are checking this in a strict way (`seller is False`), so if no substantial benefit from changing this, then don't do it for avoiding potential problems. I know that other modules while being migrated can change this, but save pain if not needed.

Reply:
About your PyCharm error, is due to the IDE, not because of the code. Python regular linters are not used to ORM framework.

Reply:
Ok.

> I know that other modules while being migrated can change this, but save pain if not needed.

But it's not a pain, it's just improving code. And also gives entertainment, because migrations are very boring. 😜

Reply:
If you promise you'll check all the dependent modules, then keep the change.